recipes/scippnexus/recipe.yaml

```diff
@@ -33,7 +33,6 @@ tests:
   - python:
       imports:
         - scippnexus
-      pip_check: false
```

Comment:
I'm not sure of true is default in v1. Maybe let's make it explicit here just to be sure?

Reply:
From the build logs

```
 │ │ Installing test environment
 │ │ ✔ Successfully updated the test environment
 │ │ + python $SRC_DIR/conda_build_script.py
 │ │ ✔ python imports test passed!
 │ │ + pip check
 │ │ No broken requirements found.
 │ │ ✔ pip check passed!

```